.travis.yml

```diff
@@ -18,7 +18,7 @@ env:
   -JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 
 script:
-  - sbt -java-home /usr/lib/jvm/java-8-openjdk-amd64 ++$TRAVIS_SCALA_VERSION test mimaReportBinaryIssues
+  - sbt -java-home /usr/lib/jvm/java-8-openjdk-amd64 "++$TRAVIS_SCALA_VERSION test" "++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues"
```

Comment:
`++XYZ` as a single command will switch Scala versions only in modules that are allowed but when running `test` afterwards the aggregation will then still run in all projects, some of the wrong scala version.

`++XYZ task` changes only in allowed modules but then also only aggregates over the allowed ones.